Drop usage of c10/util/string_view.h

kaiyuan-li opened the pull request:
Differential Revision: D57209935

kaiyuan-li commented:
I have fixed a few errors previously in https://github.com/pytorch/pytorch/pull/107147?fbclid=IwAR2g-SXHwRT5pzG2aTCYxJU4wBZLdKX4g-dp6yRSS5qEhm3iETBJthz-oDc

aaronenyeshi reviewed (changes requested):
Please check the failures!

kaiyuan-li committed:
Drop usage of c10/util/string_view.h (#125916)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/125916

Differential Revision: D57209935
